3.6.1/entrypoint.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+log(){
+	while read line ; do
+		echo "`date '+%D %T'` $line"
+	done
+}
+
+set -e
+logfile=/home/LogFiles/entrypoint.log
+test ! -f $logfile && mkdir -p /home/LogFiles && touch $logfile
+exec > >(log | tee -ai $logfile)
+exec 2>&1
+
+set_var_if_null(){
+	local varname="$1"
+	if [ ! "${!varname:-}" ]; then
+		export "$varname"="$2"
+	fi
+}
+
+python --version
+pip --version
+
+echo "INFO: starting SSH ..."
+service ssh start
+
+# setup nginx log dir
+test ! -d "$NGINX_LOG_DIR" && echo "INFO: $NGINX_LOG_DIR not found, creating ..." && mkdir -p "$NGINX_LOG_DIR"
```

Comment:
why we’re logging in the current way, because we found everything we echoed didn’t go into the docker_out logs when we were trying to fix issues for WordPress docker image. Someone said he didn’t see anything from the docker_out.log. So we specified /home/LogFiles/entrypoint.log as the log file in the lines at the top of entrypoint.sh, and then we fixed the issues by detecting /home/LogFiles/entrypoint.log.

but okay. we can try to remove these lines to see if docker out logs work as expected.